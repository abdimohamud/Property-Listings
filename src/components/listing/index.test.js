import * as React from 'react'
import renderer from 'react-test-renderer'

import Listing from '.'

test('renders correctly', () => {
  const listing = {
    id: 1,
    image: 'image.png',
    title: 'title',
    address: 'address',
    description: 'description',
    price: '999'
  }
  const tree = renderer.create(<Listing listing={listing} />).toJSON()
  expect(tree).toMatchSnapshot()
})
