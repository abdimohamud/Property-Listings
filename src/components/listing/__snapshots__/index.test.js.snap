// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="column col-4 col-xs-12 column"
>
  <div
    className="card card"
  >
    <div
      className="card-image"
    >
      <img
        alt="address"
        className="img-responsive"
        src="/server/image.png"
      />
    </div>
    <div
      className="card-header"
    >
      <div
        className="card-title h5"
      >
        title
      </div>
      <div
        className="card-title h6"
      >
        Â£
        999
      </div>
      <div
        className="card-subtitle text-gray"
      >
        address
      </div>
    </div>
    <div
      className="card-body"
    >
      description
    </div>
    <div
      className="card-footer"
    >
      <a
        className="btn btn-primary"
        href="/details/1"
        onClick={[Function]}
      >
        Go to property
      </a>
    </div>
  </div>
</div>
`;
